src/AttrTransform/ImgRequired.hack

```diff
@@ -0,0 +1,50 @@
+// must be called POST validation
```

Comment:
https://github.com/ezyang/htmlpurifier/blob/master/library/HTMLPurifier/AttrTransform/ImgRequired.php is original PHP